model/rulefmt/rulefmt.go

```diff
@@ -58,10 +63,14 @@ type WrappedError struct {
 
 // Error prints the error message in a formatted string.
 func (we *WrappedError) Error() string {
+	if we.err == nil {
+		return ""
+	}
 	if we.nodeAlt != nil {
-		return errors.Wrapf(we.err, "%d:%d: %d:%d", we.node.Line, we.node.Column, we.nodeAlt.Line, we.nodeAlt.Column).Error()
-	} else if we.node != nil {
-		return errors.Wrapf(we.err, "%d:%d", we.node.Line, we.node.Column).Error()
+		return fmt.Sprintf("%d:%d: %d:%d: %v", we.node.Line, we.node.Column, we.nodeAlt.Line, we.nodeAlt.Column, we.err)
```

Comment:
```suggestion
		return fmt.Sprintf("%d:%d: %d:%d: %w", we.node.Line, we.node.Column, we.nodeAlt.Line, we.nodeAlt.Column, we.err)
```

Reply:
Are you sure ? I have this warning in vscode : 
> fmt.Sprintf does not support error-wrapping directive %w